Improve emote bugs page

Andre601 opened the pull request:
This improves the emotes bug page in a few ways.

It first of all moves the content to a more generic named page (`broken-emote-textures.md`) as the current one implies it to only be an issue with shaders, which is completely wrong.

Next does it also add "Customizable Player Models" as an incompatible mod to the list.

Finally is it splitting up the page into Shaders and Mods that can/will change the player model.

To not break existing links did I keep the old page, but made it link to the new one... I hope the page-ref thing works.

LoneDev6 commented:
Is it really necessary to have separate this information in two pages?
There is already too much nesting in the wiki

Andre601 commented:
> Is it really necessary to have separate this information in two pages?
> There is already too much nesting in the wiki 

It isn't separated. Just moved and the old page kept to not break existing links.

I moved the page as it is very confusing to have non-shader mods mentioned in a page talking about emotes being broken by shader mods.
